shared/pub/metadata.js

```diff
@@ -0,0 +1,54 @@
+export const getPDFDownloads = (pub) => {
+	const downloads = pub.downloads;
+	const exports = pub.activeBranch.exports;
+	if (downloads && downloads.length > 0) {
+		return downloads.filter((download) => {
+			if (download.url.substring(download.url.length - 4) === '.pdf') {
+				return download;
+			}
+			return false;
+		});
+	}
+	if (exports && exports.length > 0) {
```

Comment:
⚠️ I think this logic needs to be reworked so that if `downloads.length > 0` but we don't find a `.pdf` download, we fall through to the exports array here. Something like:

```js
if (downloads) {
    const matchingDownload = downloads.find(dl => dl.url.endsWith('.pdf');
    if (matchingDownload) return matchingDownload;
}
if (exports) {
   // ...
}
```